demo/admin/src/products/future/generated/CombinationFieldsTestProductsGrid.tsx

```diff
@@ -317,6 +316,113 @@ export function ProductsGrid({ toolbarAction, rowAction, actionsColumnWidth = 52
             flex: 1,
             minWidth: 150,
         },
+        {
+            field: "nestedGroups",
+            headerName: intl.formatMessage({ id: "product.nestedGroups", defaultMessage: "Nested groups" }),
+            filterable: false,
+            sortable: false,
+            renderCell: ({ row }) => {
+                const secondaryTextNestedItem3GroupValues: string[] = [
+                    typeof row.price === "undefined" || row.price === null
```

Comment:
I'd use `row.price == null` here, which covers both undefined and null.

Reply:
This was implemented discussed here: https://github.com/vivid-planet/comet/pull/2491#discussion_r1752059796

> I don't really care, just thought more explicit would be slightly better 

This is still my opinion on this 🙂